inc/class_parser.php

```diff
@@ -1357,7 +1357,13 @@ function mycode_parse_video($video, $url)
 		switch($video)
 		{
 			case "dailymotion":
-				list($id) = explode('_', $path[2], 2); // http://www.dailymotion.com/video/fds123_title-goes-here
+				if(isset($path[2])){
```

Comment:
Could you shift the `{` to a new line here please?

Reply:
Oops! Done. Sorry ...